.eslintrc

```diff
@@ -77,6 +77,14 @@ rules:
   ## Suggest using 'const' wherever possible
   prefer-const: 2
 
+  # Arrow Functions
+  ## Enforce parens around arrow function arguments
+  arrow-parens: [2, "always"]
+  ## Require a space on each side of arrow operator
+  arrow-spacing: [2, {"before": true, "after": true}]
+  ## Prevent using => in a condition where <= is intended
+  no-arrow-condition: 2
+
```

Comment:
There's not `Arrow Functions` category in http://eslint.org/docs/rules/. Can you structure the rules like in that list?